portfolio/src/main/webapp/script.js

```diff
@@ -78,15 +117,51 @@ const appendComments = (comments, commentContainerElement) => {
     const commentElement =  document.createElement("div");
     commentElement.classList.add("comment");
 
+    const trashIconElement = document.createElement("i");
+    trashIconElement.classList.add("fa");
+    trashIconElement.classList.add("fa-trash");
+    trashIconElement.setAttribute("area-hidden", "true");
+
+    const trashElement = document.createElement("a");
+    trashElement.classList.add("trash-icon");
+    const deleteFunctionParameter = "deleteComment(" + comments[commentId].id + ")";
+    trashElement.setAttribute("onClick", deleteFunctionParameter);
+    trashElement.setAttribute("alt", "Delete comments");
+    trashElement.appendChild(trashIconElement);
+
     // append each element to the father element
     commentElement.appendChild(userElement);
+    commentElement.appendChild(votesContainerElement);
     commentElement.appendChild(messageElement);
     commentElement.appendChild(datePostedElement);
+    commentElement.appendChild(trashElement);
 
     // Make a final append to the comment container element
     commentContainerElement.appendChild(commentElement);
   });
 };
 
+/**
+* Deletes a comment given the id of the comment
+* @param {long} commentId - the id of the comment that will be deleted.
+*/
+const deleteComment = (commentId) => {
```

Comment:
Why are you storing the functions as const variables? (Pure curiosity)

Reply:
It is a way to declare them on _ES6_, you can use either **const** or **let**, in this example I use **const** because I won't be **override** the function in other part of my code.